ale/base/base.py

```diff
@@ -1,8 +1,7 @@
 import pvl
 import abc
-from abc import ABC
 
-class Driver(ABC):
```

Comment:
Does removing inheritance from ABC stop the abstract property tags from throwing an exception if the method gets called without being replaced?

Reply:
Everything changed with notimplemented error so it should still throw an error.